packages/allure-cucumberjs/README.md

```diff
@@ -13,10 +13,16 @@ export default class Reporter extends CucumberJSAllureFormatter {
       options,
       new AllureRuntime({ resultsDir: "./allure-results" }),
       {
-        labels: {
-          epic: [/@feature:(.*)/],
-          severity: [/@severity:(.*)/]
-        },
+        labels: [
+          {
+            pattern: [/@feature:(.*)/],
+            type: "epic"
```

Comment:
maybe use `name` instead of `type` (there is no such thing as label type)?

Reply:
👍